app/talk/comment-box.cjsx

```diff
@@ -118,7 +129,12 @@ module?.exports = React.createClass
               {if @state.showing is 'image-selector' then <span>&nbsp;<i className="fa fa-close" /></span>}
             </button>
 
-          <SingleSubmitButton type="submit" onClick={@onSubmitComment} className='talk-comment-submit-button'>{@props.submit}</SingleSubmitButton>
+            <input type="checkbox" className="honey-input" tabindex="-1" name="agree-to-terms" value="agree-to-terms" ref="honeyCheck"/>
+            <input type="text" className="honey-input" tabindex="-1" name="better-title" ref="honeyText" />
+            <button type="submit" className="honey-input" tabindex="-1" onClick={@onFakeSubmit}>Submit</button>
```

Comment:
Multiple `type="submit"` buttons are ambiguous, and currently a fake one swallows the enter key.

Does the submit button they clicked actually reveal anything about the humanness of the user anyway?